common/scala/Dockerfile

```diff
@@ -19,7 +19,7 @@ ENV PATH $JAVA_HOME/bin:$PATH
 
 
 RUN apk upgrade --update && \
-    apk add --update libstdc++ curl ca-certificates bash sed && \
+    apk add --update libstdc++ curl ca-certificates bash sed wget&& \
```

Comment:
Hum no problem with not having a space in wget&& ?

Reply:
Looks like in hurry I forgot to add a space. However it does works and [wget gets updated](https://travis-ci.org/apache/incubator-openwhisk/jobs/386053710#L1980)

```
(12/13) Installing sed (4.4-r0)
(13/13) Installing wget (1.19.5-r0)
Executing busybox-1.26.2-r11.trigger
Executing ca-certificates-20161130-r2.trigger
```
Would fix this in a new PR

Reply:
Opened PR #3723